avm/res/db-for-postgre-sql/flexible-server/tests/e2e/public/main.test.bicep

```diff
@@ -154,5 +154,9 @@ module testDeployment '../../../main.bicep' = [
         Role: 'DeploymentValidation'
       }
     }
+    dependsOn: [
+      nestedDependencies
+      diagnosticDependencies
+    ]
```

Comment:
```suggestion
```

Unfortunately, it turned out this doesn't help. We originally added this across the board as sometimes, deployments would not wait for their dependencies. Turned out, it's actually not helping and it's a caching issue of the backend. Waiting for some time before a retry of a deployment resolves these issues.

Reply:
once committed, we can merge the PR in. A re-run is not required.